Gloss/Gloss/Decoder.swift

```diff
@@ -202,4 +202,31 @@ public struct Decoder {
         }
     }
     
+    /**
+     Returns function to decode JSON to URL array
+     
+     :parameter: key JSON key used to set value
+     
+     :returns: Function decoding JSON to an optional URL array
+     */
+    public static func decodeURLArray(key: String) -> JSON -> [NSURL]? {
+        return {
+            json in
+            
+            if let urlStrings = json[key] as? [String] {
+                var urls: [NSURL] = []
+                
+                for urlString in urlStrings {
```

Comment:
I know this is an academic discussion when it comes to Swift or Obj-C, but here I go: Fast enumeration does not guarantee iteration order. (Swift docs don't say this, neither do Obj-C).